gradle-examples/composite-build/settings.gradle

```diff
@@ -0,0 +1,7 @@
+rootProject.name = 'thorntail-composite-app'
+
+// Include the dependent platform projects if they are around.
+File platformPrj = new File(rootDir.parentFile, 'multi-module')
+if(platformPrj.exists()) {
+    includeBuild '../multi-module'
```

Comment:
If you have the dependent project cloned locally, then you can use this approach for _including the project_ as part of your application's build.